dev/idm-am-common-user/idm/conf/repo.ds.json

```diff
@@ -0,0 +1,489 @@
+                        "ldapAttribute": "fr-idm-link-type"
+                    },
+                    "linkQualifier": {
+                        "type": "simple",
+                        "ldapAttribute": "fr-idm-link-qualifier"
+                    },
+                    "firstId": {
+                        "type": "simple",
+                        "ldapAttribute": "fr-idm-link-firstId"
+                    },
+                    "secondId": {
+                        "type": "simple",
+                        "ldapAttribute": "fr-idm-link-secondId"
+                    }
+                }
+            },
+            "clusteredrecontargetids": {
+                "dnTemplate": "ou=clusteredrecontargetids,dc=openidm,dc=example,dc=com",
+                "objectClasses": [
+                    "uidObject",
+                    "fr-idm-recon-clusteredTargetIds"
+                ],
+                "properties": {
+                    "_id": {
+                        "type": "simple",
+                        "ldapAttribute": "uid",
+                        "isRequired": true,
+                        "writability": "createOnly"
+                    },
+                    "reconId": {
+                        "type": "simple",
+                        "ldapAttribute": "fr-idm-recon-id"
+                    },
+                    "targetIds": {
+                        "type": "json",
+                        "ldapAttribute": "fr-idm-recon-targetIds"
+                    }
+                }
+            },
+            "managed/user": {
+                "dnTemplate": "ou=people,ou=identities",
+                "namingStrategy": {
+                    "type": "serverNaming",
+                    "dnAttribute": "uid",
+                    "idAttribute": "entryUUID"
+                },
+                "nativeId" : true,
+                "objectClasses": [
+                    "person",
+                    "organizationalPerson",
+                    "inetOrgPerson",
+                    "iplanet-am-user-service",
+                    "devicePrintProfilesContainer",
+                    "kbaInfoContainer",
+                    "fr-idm-managed-user-explicit"
+                ],
+                "properties": {
+                    "_id": {
+                        "type": "simple",
+                        "ldapAttribute": "entryUUID",
+                        "writability": "createOnly"
+                    },
+                    "userName": {
+                        "primaryKey": true,
+                        "type": "simple",
+                        "ldapAttribute": "uid"
+                    },
+                    "cn": {
+                        "type": "simple",
+                        "ldapAttribute": "cn"
+                    },
+                    "password": {
+                        "type": "simple",
+                        "ldapAttribute": "userPassword"
+                    },
+                    "aliasList": {
```

Comment:
Note that this attribute isn't being maintained by the managed service. At one point I had code in there which computed this value based on the various IDP relationships, but I removed that in later versions of my sample since I was no longer focusing on social registration / login. Likely we will want to restore that calculation so that this value can be available to AM in the format it needs.

Reply:
I am going to remove this for now. We can add it back when we validate that usecase.